app/views/shared/_related_items.html+taxonomy_navigation.erb

```diff
@@ -1,15 +0,0 @@
-<div class="column-third add-title-margin">
-  <% if @navigation.should_present_taxonomy_navigation? %>
```

Comment:
The observant reader will notice that because this partial variant will only be shown when `should_present_taxonomy_navigation?` is true, this statement will also always be true. The `else` could never have been executed.